include/oneapi/dpl/internal/common_config.h

```diff
@@ -24,6 +24,8 @@
 #        ifndef _GLIBCXX_USE_TBB_PAR_BACKEND
 #            define _GLIBCXX_USE_TBB_PAR_BACKEND (_GLIBCXX_RELEASE > 10)
 #        endif
+#    elif !_ONEDPL_PAR_BACKEND_TBB && !defined(PSTL_USE_PARALLEL_POLICIES)
+#        define PSTL_USE_PARALLEL_POLICIES (_GLIBCXX_RELEASE != 9)
```

Comment:
Are we sure that `_ONEDPL_PAR_BACKEND_TBB` is defined when this header is included? Maybe it's better to just check for absence of TBB headers (`tbb.h`) here.

Reply:
The `_ONEDPL_PAR_BACKEND_TBB` check  is redundant.